sdk/keyvault/azure-keyvault-keys/README.md

```diff
@@ -135,11 +149,11 @@ key_client = KeyClient(vault_url="https://my-key-vault.vault.azure.net/", creden
 
 ## Key concepts
 ### Keys
-Azure Key Vault can create and store RSA and elliptic curve keys. Both can
+Azure Key Vault can create and store RSA, elliptic curve, and octet sequence (symmetric) keys. All can
```

Comment:
You must mean "Azure Key Vault Managed HSM" 😜

Reply:
Yeah, this is probably a good illustration of how switching between KV-general and HSM-specific features in the readme might me a bit too confusing. `KeyType` makes oct keys discoverable and service documentation is pretty clear about symmetric key use, so I'll just leave in the minor changes to existing content instead of muddying the waters with MHSM speak